src/db/models/bookmark.js

```diff
@@ -0,0 +1,43 @@
+export default (sequelize, DataTypes) => {
+  const Bookmark = sequelize.define('Bookmark', {
+    userId: {
+        primaryKey: true,
+        type: DataTypes.INTEGER,
+        allowNull: {
+            args: false,
+            message: 'User id must is required'
+        }
+    },
+    slug: {
```

Comment:
you don't need two primary keys.

Reply:
I used two primary key because you can have the same `userId` and same `slug` but you can not have same as both

Reply:
and if you add the primary key in your model it creates a primary of 2 attributes in the database

Reply:
you can't have two primary keys in a db. Please fix it

Reply:
Okay